pkgs/by-name/cg/cgterm/package.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, SDL
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "cg-term";
+  version = "1.7b2";
+
+  src = fetchFromGitHub {
+    owner = "MagerValp";
+    repo = "CGTerm";
+    rev = finalAttrs.version;
+    hash = "sha256-Gk7t9wnVCRWwnqcItS3j031VqJnBqk6rHw1SABtzqfE=";
+  };
+
+  buildInputs = [
+    SDL
+  ];
+
+  makeFlags = [ "PREFIX=$(out)" ];
+
+  meta = with lib; {
+    description = "C/G telnet client for C64 BBS's";
+    homepage = "https://github.com/MagerValp/CGTerm";
+    license =  {
+      fullName  = "CGTerm";
+      url = "github.com/MagerValp/CGTerm/blob/01e35d64c29bccee52211b0afc66035a10e4792a/cgterm.txt";
```

Comment:
The license included in the link is just the regular [BSD-2-Clause](https://spdx.org/licenses/BSD-2-Clause.html).

Reply:
Thanks, not a license guru myself, so didn't know :+1: